README.md

```
@@ -259,3 +260,13 @@ To contribute to the project, get started by reading our [Contributing Guide](ht
 - Blog: https://www.qodo.ai/blog/
 - Troubleshooting: https://www.qodo.ai/blog/technical-faq-and-troubleshooting/
 - Support: support@qodo.ai
+
+
+## run pr-agent locally
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)